docs/content/preview/yugabyte-platform/security/create-kms-config/google-kms.md

```diff
@@ -0,0 +1,98 @@
+  - /preview/yugabyte-platform/security/create-kms-config
+menu:
+  preview_yugabyte-platform:
+    parent: security
+    identifier: create-kms-config-1-google-kms
+    weight: 27
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="{{< relref "./aws-kms.md" >}}" class="nav-link">
+      <i class="fab fa-aws" aria-hidden="true"></i>
+      AWS KMS
+    </a>
+  </li>
+  <li >
+    <a href="{{< relref "./google-kms.md" >}}" class="nav-link active">
+      <i class="icon-google" aria-hidden="true"></i>
+      Google KMS
+    </a>
+  </li>
+
+  <li >
+    <a href="{{< relref "./azure-kms.md" >}}" class="nav-link">
+      <i class="icon-azure" aria-hidden="true"></i>
+      &nbsp;&nbsp;&nbsp;&nbsp;Azure KMS
+    </a>
+  </li>
+
+  <li >
+    <a href="{{< relref "./hashicorp-kms.md" >}}" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      HashiCorp Vault
+    </a>
+  </li>
+
+</ul>
+
+Encryption at rest uses universe keys to encrypt and decrypt universe data keys. You can use the YugabyteDB Anywhere UI to create key management service (KMS) configurations for generating the required universe keys for one or more YugabyteDB universes. Encryption at rest in YugabyteDB Anywhere supports the use of Google Cloud KMS. 
+
+Conceptually, Google Cloud KMS consists of a key ring containing one or more cryptographic keys, with each key capable of having multiple versions.
+
+Before defining a KMS configuration with YugabyteDB Anywhere, you need to create a key ring through [Google Cloud KMS](https://cloud.google.com/security-key-management).
+
+The Google Cloud user associated with a KMS configuration requires a custom role assigned to the service account with the following KMS-related permissions:
+
+- `cloudkms.keyRings.create`
+- `cloudkms.keyRings.get`
+- `cloudkms.keyRings.list`
+- `cloudkms.cryptoKeys.create`
+- `cloudkms.cryptoKeys.get`
+- `cloudkms.cryptoKeys.list`
+- `cloudkms.cryptoKeyVersions.useToEncrypt`
+- `cloudkms.cryptoKeyVersions.useToDecrypt`
+- `cloudkms.cryptoKeyVersions.destroy`
+- `cloudkms.locations.generateRandomBytes`
+
+If you are planning to use an existing cryptographic key with the same name, it must meet the following criteria:
+
+- The primary cryptographic key version should be in the Enabled state.
+- The purpose should be set to symmetric ENCRYPT_DECRYPT.
+- The key rotation period should be set to Never (manual rotation).
+
+You can create a KMS configuration that uses Google Cloud KMS, as follows:
+
+1. Use the YugabyteDB Anywhere UI to navigate to **Configs > Security > Encryption At Rest** to access the list of existing configurations.
+
+1. Click **Create New Config**.
+
+3. Enter the following configuration details in the form:
+
+    - **Configuration Name** — Enter a meaningful name for your configuration.
+    - **KMS Provider** — Select **GCP KMS**.
+    - **Service Account Credentials** — Download the JSON file containing your Google Cloud credentials.
```

Comment:
Shouldn't this be "Upload the JSON file..."?

Reply:
Fixed